commerce_coordinator/apps/commercetools/pipeline.py

```diff
@@ -87,3 +90,51 @@ def run_filter(self, params, active_order_management_system, order_number):  # p
         except CommercetoolsError as err:  # pragma no cover
             log.exception(f"[{type(self).__name__}] Commercetools Error: {err}, {err.errors}")
             return PipelineCommand.CONTINUE.value
+
+
+class CreateReturnForCommercetoolsOrder(PipelineStep):
+    """
+    Creates refund/return for Commercetools order by Updating its
+    ReturnShipmentStatus & ReturnPaymentStatus
+    """
+
+    def run_filter(
+            self,
+            active_order_management_system,
+            order_number,
+            order_line_id
+            ):  # pylint: disable=arguments-differ
+        """
+        Execute a filter with the signature specified.
+        Args:
+            request: django.http.HttpRequest object passed through from lms views.py
+        Returns:
+            returned_order: Updated Commercetools order
+            returned_order_line_item_id: Updated Commercetools order's line item id
+
+        """
+        if active_order_management_system != COMMERCETOOLS_ORDER_MANAGEMENT_SYSTEM:
+            return PipelineCommand.CONTINUE.value
+
+        try:
+            ct_api_client = CommercetoolsAPIClient()
+            order = ct_api_client.get_order_by_id(order_id=order_number)
+
+            if not is_commercetools_line_item_already_refunded(order, order_line_id):
+                returned_order = ct_api_client.create_return_for_order(
+                    order_id=order.id,
+                    order_version=order.version,
+                    order_line_id=order_line_id
+                )
+
+                return {
+                    'returned_order': returned_order,
+                    'returned_order_line_item_id': order_line_id
```

Comment:
You might also need to return the order's return item's ID as @JadeyOlivier's [PR](https://github.com/edx/commerce-coordinator/pull/172/files#diff-d1a78066c20090f2c1df68b04f57ddb52bbd0886764ab53603465b23c4677e67R292) is consuming that ID to update the return item's payment state.

Reply:
Correct yes